lib/Sema/MiscDiagnostics.cpp

```diff
@@ -1184,6 +1184,39 @@ bool swift::fixItOverrideDeclarationTypes(TypeChecker &TC,
     return true;
   };
 
+  // Check if overriding fails because we lack @escaping attribute on the function
+  // type repr.
+  auto checkTypeMissingEscaping = [&](Type overrideTy, Type baseTy,
+                                      SourceRange typeRange) -> bool {
+    // Fix-it needs position to apply.
+    if (typeRange.isInvalid())
+      return false;
+    auto overrideFnTy = overrideTy->getCanonicalType()->getAs<FunctionType>();
```

Comment:
Overriding isn't *this* strict. This doesn't catch
- covariant return types / contravariant parameters
- optionality changes for class types
- anything involving generics

(Also, the `getCanonicalType` isn't necessary; `getAs` looks through things.)